lib/app-service-registration.js

```diff
@@ -20,6 +20,7 @@ function AppServiceRegistration(appServiceUrl) {
         aliases: [],
         rooms: []
     };
+    this.protocols = [];
```

Comment:
I _think_ this should be null, just so we don't get an empty list in new reg files.